config/serializer/ads/Node.yml

```diff
@@ -34,3 +34,6 @@ Adshares\Ads\Entity\Node:
         status:
             expose: true
             type: int
+        version:
+            expose: true
+            type: string
```

Comment:
Node entity in ADS Client does not have such field. We can remove it from here.

Reply:
Removed